File: spec/code_samples/Javascript/user@email_transfers/post.js
'use strict';
const ccxt = require ('ccxt');

(async function () {
  let paymium = new ccxt.paymium({
    apiKey: 'YOUR_PUBLIC_API_KEY',
    secret: 'YOUR_SECRET_PRIVATE_KEY',
  })

  console.log(paymium.id, await paymium.privatePostUserEmailTransfers({ "currency": "BTC", "amount": "1.0", "email": "johnsmith@example.com", "comment": "For your savings" }))
})();

File: spec/code_samples/Python/merchant@create_payment/post.py
# coding=utf-8

import ccxt

paymium = ccxt.paymium({
  'apiKey': 'YOUR_PUBLIC_API_KEY',
  'secret': 'YOUR_SECRET_PRIVATE_KEY',
})

print(paymium.privatePostMerchantCreatePayment({
  "amount": "0.0001",
  "payment_split": "0",
  "currency": "BTC",
  "callback_url": "https://myonlineshop/payments/order-987978/callback",
  "redirect_url": "http://myonlineshop/payments/order-987978/success",
  "merchant_reference": "order-987978"
}))